tiledb/sm/compressors/rle_compressor.cc

```diff
@@ -145,6 +145,19 @@ uint64_t RLE::overhead(uint64_t nbytes, uint64_t value_size) {
   return value_num * 2;
 }
 
+uint8_t RLE::compute_bytesize(uint64_t param_length) {
+  assert(param_length > 0);
+  if (param_length <= std::numeric_limits<uint8_t>::max()) {
+    return 1;
+  } else if (param_length <= std::numeric_limits<uint16_t>::max()) {
+    return 2;
+  } else if (param_length <= std::numeric_limits<uint32_t>::max()) {
+    return 3;
```

Comment:
should this be 4?

Reply:
yes! thanks for catching this. fixing

Reply:
I will add UTs for those functions in my next PR with the rest of the tests I am writing.